blivet/devicelibs/dasd.py

```diff
@@ -144,6 +144,30 @@ def is_ldl_dasd(device):
         return False
     return True
 
+def is_fba_dasd(device):
+    """Is the device a FBA DASD?"""
+    if not device.startswith("dasd"):
+        # not a dasd; bail
+        return False
+
+    device = "/dev/%s" % (device,)
```

Comment:
I know it will not matter in this case, but I would prefer if you passed in the full path instead of the name so the device class can handle building the full path.